docs/cip/README.md

```diff
@@ -4,704 +4,2873 @@ Title: Ouroboros Leios - Greater transaction throughput
+  expires, which can be up to 90 days later on Cardano mainnet. That's
+  unacceptably long. (Significantly shortening the 90 day period is not an
+  option, because that would excessively burden SPOs by forcing them to utilize
+  their cold key more often.)
+- If Leios instead over-interprets distinct OCINs as separate elections, then
+  any adversary can diffuse any number of EB announcements per actual election,
+  with arbitrary OCINs. Those announcements would be an unacceptable unbounded
+  burst of work for honest nodes to relay throughout the entire network, even if
+  they still only relayed at most one EB body per actual election.
+
+There is no simple compromise between those extrema. In particular, if there's
+any way for the adversary to have revealed incremented OCINs to some nodes but
+definitely not all, then the worst-case diffusion behavior of adversarial EBs
+might be worse than that of honest EBs, which would complicate the acceptable
+lower bound on $L_\text{diff}$, for example. On the other hand, if every OCIN
+increment - even those disallowed by Praos - is always eventually relayed to all
+nodes, then an adversary can create unbounded work by constantly incrementing
+their OCINs - without the context provided by forks, there's no bound on the
+OCINs the Leios mini-protocols would relay.
+
+Thus, the diffusion of EB announcements (regardless of who issued them) is only
+tractable and robust if it is restricted to a bounded set of OCINs that all
+honest nodes almost certainly agree on. For this reason, the Leios node should
+ignore an EB announcement that is less than the OCIN on its settled ledger state
+or more than one greater than that OCIN. After not ignoring two announcements
+with the same election, the Leios node should ignore (including not relaying)
+any subsequent announcements for that election. An intended implication of this
+rule is that an honest server would send the same one or two announcements to
+all of its clients; it does not have to track any extra state per-client.
+
+With this rule, a client will crucially disconnect if and only if a server sends
+more than two announcements with the same election. It will also ignore headers
+from leaked hot keys once the SPO increments their OCIN, but unfortunately - and
+in contrast to Praos - not immediately. The Leios node will only ignore
+unincremented OCINs after the increment has settled on the chain. With Ouroboros
+Peras, that could be a matter of minutes. Without Peras, it is typically 12 hr
+under normal conditions and would take at most 36 hr before Cardano would
+consider it to be a disaster scenario. Thus, if an SPO leaks their hot key, the
+network would not diffuse their EBs until up to 36 hr later (usually 12 hr, and
+much sooner with Ouroboros Peras). SPOs should not leak their hot keys that
+frequently, and even if they do, it is still much sooner than waiting for the
+full 90 days.
+
+### Incentives
```

Comment:
I'm really struggling to understand what we are _actually proposing_ / specifying in terms incentives and what is a discussion of what could be done or should even be avoided!?